GitHeart/MemoryCache/MemoryCache.swift

```diff
@@ -24,6 +25,9 @@ protocol ByteSizable {
 /// in a best case is O(1).
 final class MemoryCache<Key: Hashable, Value: ByteSizable> {
     private let serialQueue = DispatchQueue(label: "MemoryCache: \(UUID().uuidString)")
+  // Why it's an array, and not an ordered dictionary where key is used only once?
+  // It would be faster to retrieve a value from it.
+  // OrderedDictionary is available in https://github.com/apple/swift-collections repo
```

Comment:
Swift collections can be considered as external dependency and I can't use external dependencies on this project.